reference/library/src/morpheus-master/sass/base/_icons.scss

```diff
@@ -71,6 +71,7 @@
   .icon-sakai--sakai-sitestats-admin {                      @extend .fa-pie-chart; }
   .icon-sakai--sakai-su {                                   @extend .fa-user-secret; }
   .icon-sakai--sakai-syllabus {                             @extend .fa-map-o; }
+  .icon-sakai--sakai-tagservice {             			    @extend .fa-tag; }
```

Comment:
I've needed to edit it here in git directly. It still was looking correctly indented in my computer..